mlflow/h2o.py

```diff
@@ -0,0 +1,80 @@
+"""Sample MLflow integration for h2o."""
+
+from __future__ import absolute_import
+
+import os
+import h2o
```

Comment:
Can we make this dependency optional by moving the import into just the functions that need it? This is what we've been doing for most other ML libraries so that MLflow doesn't end up with a huge number of dependencies and a slow installation process.